### DISCUSSION TITLE
Is there any official strategy to deal with sandstorms?

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain#11794] [General]
Is there any official strategy to deal with sandstorms? More and more wallets are currently affected.

**Comment by stefan-lange** [comment]
Seems not, it also pollutes the blockchain db and costs energy for nothing. The only chance to reduce them is to set sth. like a minimal transaction cost or minimal height of a transaction. But this would lead to problems with the faucet

**Comment by stefan-lange** [comment]
Something is 'going on'. No concrete solution so far, but the awareness: https://github.com/Chia-Network/chips/blob/9baab1dc92e4819d016f27bab8311de1ce2acc77/CHIPs/chip-0003.md
